lambda_demo/src/main/java/org/example/s3lambda/S3EventHandler.java

```diff
@@ -0,0 +1,75 @@
+package org.example.s3lambda;
+
+// To build: mvn clean compile assembly:single
```

Comment:
Usually we indicate building info on README file

Reply:
fixed